src/lib/minions/types.ts

```diff
@@ -65,6 +65,7 @@ export interface KillableMonster {
 	respawnTime?: number;
 	levelRequirements?: LevelRequirements;
 	uniques?: ArrayItemsResolved;
+	consumedItem?: ArrayItemsResolved;
```

Comment:
i would like this to be a bank both for future proofing where more than one item can be consumed and for managing that consumption you wouldnt have to convert this to a bank to remove it from the players